lstchain/reco/r0_to_dl1.py

```diff
@@ -220,14 +220,14 @@ def r0_to_dl1(
 
     custom_calibration = config["custom_calibration"]
 
-    source_config={
+    source_config = {
         "LSTEventSource": {
             "allowed_tels": [1],
             "EventTimeCalculator": {
-                "ucts_t0_dragon": int(ucts_t0_dragon),
-                "dragon_counter0": int(dragon_counter0),
-                "ucts_t0_tib": int(ucts_t0_tib),
-                "tib_counter0": int(tib_counter0)
```

Comment:
Hi @maxnoe,

if we do not set this as integer we get the error already mentioned by @moralejo